platform/authorized_api/apps/index.json

```diff
@@ -385,6 +402,12 @@
             "*.example.com",
             "example.com"
           ]
+        },
+        "gdprLocationRecognition": {
+          "type": "boolean",
+          "description": "Use geoIP to detect UE visitors and present consent forms only to them",
```

Comment:
We fixed this description a little bit so here should be `geolocation` instead of `geoIP`